src/box/wal_ext.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+#include "trivia/config.h"
+#include "say.h"
+
+#if defined(ENABLE_WAL_EXT)
+# include "wal_ext_impl.h"
+#else /* !defined(ENABLE_WAL_EXT) */
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/** No-op in OS version. */
+static inline void
+wal_ext_free(void)
+{
+}
```

Comment:
`wal_ext_free` without `wal_ext_init` looks weird. Please add `wal_ext_init`. You can initialize the hash table that stores per space config there.

Reply:
Ok, added another one stub.